src/transports/ws/bws.c

```diff
@@ -314,6 +344,71 @@ static void nn_bws_handler (struct nn_fsm *self, int src, int type,
         }
 
 /******************************************************************************/
+/*  CLOSING_USOCK state.                                                      */
```

Comment:
Agree here. I'll leave the format for now for conformance, then we can come back later and change them all in bulk.